cabby/geo/map_processing/map_structure.py

```diff
@@ -339,9 +344,9 @@ def add_two_ways_edges(self, edge_add: edge.Edge):
 
   def add_poi_to_graph(self):
     '''Add all POI to nx_graph(currently contains only the roads).'''
-    eges_to_add_list = self.poi.apply(self.add_single_poi_to_graph, axis=1)
-    
-    eges_to_add_list.swifter.apply(
+    edges_to_add_list = self.poi.apply(self.add_single_poi_to_graph, axis=1)
+    edges_to_add_list = edges_to_add_list.dropna()
```

Comment:
In what cases is this needed?

Reply:
In this PR, `add_single_poi_to_graph` now returns `None` if the POI is already in the graph. So this removes any entries in the series `edges_to_add_list` that are `None`.